app/code/AndriiShkrebtii/RegularCustomer/Setup/Patch/Data/RemoveUniqueIndex.php__

```diff
@@ -0,0 +1,49 @@
+<?php
+
+declare(strict_types=1);
+
+namespace AndriiShkrebtii\RegularCustomer\Setup\Patch\Data;
+
+class RemoveUniqueIndex implements \Magento\Framework\Setup\Patch\SchemaPatchInterface
+{
+    /**
+     * @var \Magento\Framework\Setup\ModuleDataSetupInterface $schemaSetup
+     */
+    private $schemaSetup;
+
+    /**
+     * RemoveOldForeignKeys constructor.
+     * @param \Magento\Framework\Setup\SchemaSetupInterface $schemaSetup
+     */
+    public function __construct(
+        \Magento\Framework\Setup\SchemaSetupInterface $schemaSetup
+    ) {
+        $this->schemaSetup = $schemaSetup;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function apply()
+    {
+        $this->schemaSetup->getConnection()->startSetup();
+        //
```

Comment:
@jimmywilson111 Implementation missed

Reply:
As soon as possible there will be a patch to fix the removal of the constraint from database.

Reply:
@jimmywilson111 Can't accept merge request without the complete patch